docs/reference/setup/install/docker.asciidoc

```diff
@@ -234,6 +235,12 @@ chmod g+rwx esdatadir
 chgrp 0 esdatadir
 --------------------------------------------
 
+You can also run an {es} container using both a custom UID and GID. Unless you
+bind-mount each of the `config`, data` and `logs` directories, you must pass
+the command line option `--group-add 0` to `docker run`. This ensures that the user
```

Comment:
I guess my only question is what is the point of this? I only assume running as a different user is something done by security conscious orgs that don't like the idea of running containers as root. Wouldn't then subsequently adding the user to the root group defeat the purpose?

Reply:
It's mostly a convenience for users who might be experimenting with the image, or are only interested in capturing the stdout/stderr output of the container and don't want to bind-mount the `logs` directory. It also serves to document why someone might have problems running the image with a custom GID.

Note that the container user is only added to the `root` group inside the container, but files in bind-mounted volumes are still written as the user-specified GID, so everything is still OK outside of the container.